libs/frontend/ui-theme/src/css/components/prosemirror-view.postcss

```diff
@@ -34,9 +34,6 @@
 .ProseMirror pre {
   white-space: pre-wrap;
 }
-.ProseMirror-trailingBreak {
```

Comment:
I think the reason why this was removed is that it results in extra spaces int the editor that don't exist in preview/live mode. You can see the difference when you press enter in the ProseMirror editor.